test/Data.Json.JTable.StrongCheck.purs

```diff
@@ -0,0 +1,112 @@
+module Test.Data.Json.JTable.StrongCheck where
+
+import Data.Array
+import qualified Data.Array.Unsafe as AU
+import Data.Foldable
+import Data.Maybe
+import Data.Tuple
+import qualified Data.StrMap as SM
+import Debug.Trace
+
+import qualified Data.Argonaut as A
+import Data.Argonaut.Core (Json(..), jsonNull)
+import Data.Argonaut.JCursor
+import qualified Text.Smolder.HTML as H
+import Text.Smolder.Markup (Markup(..), text)
+import qualified Text.Smolder.Renderer.String (render) as Sm
+
+import Test.StrongCheck
+import Test.StrongCheck.Gen
+
+import Data.Json.JTable
+import Data.Json.JTable.Internal
+
+
+genJsonArray :: Gen Json
+genJsonArray = sized \size -> 
+  (vectorOf size $ resize (size-1) genJson) <#> A.fromArray
+
+genJsonObject :: Gen Json
+genJsonObject = sized \size -> 
+  (vectorOf size $ resize (size-1) arbitrary) <#> SM.fromList >>> A.fromObject
+
+genJson :: Gen Json
```

Comment:
This is nice. I think it may be better than the one in Argonaut because it uses the size parameter.